app/travis_job.rb

```diff
@@ -0,0 +1,57 @@
+require "securerandom"
+require "json"
+
+module Tint
+	class TravisJob
+		attr_reader :job_id, :site, :status
+
+		def initialize(site_or_payload, job_id=SecureRandom.uuid, status=nil)
+			if site_or_payload.is_a?(Tint::Site)
+				@site = site_or_payload
+			else
+				@payload = site_or_payload
+				@site = Tint::Site.new(DB[:sites][site_id: @payload['site']['site_id']])
+			end
+
+			@job_id = job_id
+			@status = status
+		end
+
+		def self.get(job_id)
+			queue = Pathname.new(ENV.fetch("TRAVIS_WORKER_BASE_DIR")).realpath
```

Comment:
We do this exact thing a lot, maybe a method? `Pathname.new(ENV.fetch("TRAVIS_WORKER_BASE_DIR")).realpath`